CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ FUNCTION(PMWCAS_DEFINE_ENV)
   if(${PMEM_BACKEND} STREQUAL "PMDK")
     add_definitions(-DPMEM)
     add_definitions(-DPMDK)
+    set(PMEM TRUE CACHE BOOL "PMEM")
     message(STATUS "Persistence support: PMDK")
   elseif(${PMEM_BACKEND} STREQUAL "EMU")
     add_definitions(-DPMEM)
```

Comment:
I think EMU also need this flag

Reply:
Oh sure. I thought we are not going to do emulation any more...